subsys/suit/platform/sdfw/src/suit_plat_invoke.c

```diff
@@ -99,7 +99,11 @@ int suit_plat_invoke(suit_component_t image_handle, struct zcbor_string *invoke_
 	switch (component_type) {
 	case SUIT_COMPONENT_TYPE_MEM:
 		/* memory-mapped */
-		ret = suit_plat_cpu_run(cpu_id, run_address);
+		ret = suit_plat_cpu_halt(cpu_id);
```

Comment:
I'd prefer to place it before `suit_event_clear` :slightly_smiling_face:

Reply:
Indeed...

Reply:
Fixed